src/initialize/src/main.ts

```diff
@@ -42,7 +42,9 @@ export async function initializeProject(projectName: string, projectDirectory: s
   const projectInitializer = new ProjectInitializer(projectName, projectDirectory);
 
   try {
-    if (isGitCloneRequired(projectName, gitRepository)) {
+    if (gitRepository.endsWith("tar.gz")) {
```

Comment:
I'm not that comfortable with overloading GIT_REPO to also mean "any url to a tar.gz". Can we just use a different env var and check if that was set or if GIT_REPO was set?